README.md

```diff
@@ -4,23 +4,28 @@ languages:
 - csharp
 products:
 - azure
-description: "The Azure Search Service Management API provides programmatic access to much of the functionality available through the Azure Search extension within the Azure Preview Management Portal."
+description: "The Azure Cognitive Search Management API provides programmatic access to create and manage your search services."
 urlFragment: search-dotnet-management-api
 ---
 
-# Azure Search .NET Management API Demo
+# Azure Cognitive Search Management .Net SDK Sample
 
-The Azure Search Service Management API provides programmatic access to much of the functionality available through the Azure Search extension within the Azure Preview Management Portal. The management API is a REST API. Its purpose is to allow administrators to automate common tasks typically performed from the Azure Portal UI.
+The Azure Cognitive Search Management API provides programmatic access to create and manage your search services.  The SDK makes it easy to automate search service management tasks typically performed from the Azure Portal UI.
 
-Examples of programmable operations include:
-- Create or delete Azure search services.
-- Alter the scale of an Azure Search service in response to changes in query scale or storage requirements. Increasing or decreasing replicas or partitions affects billing and will change the number of Search Units allocated to a service.
-- Create or change api-keys. This allows administrators to automate regular changes to administrative keys to the Azure Search service.
+ Sample for managing a search service -
+ - Register the Microsoft.Search provider in a subscription
+ - List search services a a subscription
+ - Create a new search service
+ - Manage service Api keys
+ - Scale a service up and down
+ - Delete a search service
 
-See Get started with Azure Search REST API http://msdn.microsoft.com/library/azure/dn798935.aspx for a walkthrough of sample code that demonstrates the API.
+Contact [Azure Search Contact Us](mailto:azuresearch_contact@microsoft.com) with questions about the sample.
 
-For questions, please reach out to me at Twitter http://twitter.com/liamca/
+## More information ##
 
----
+[Azure Search libraries for .NET](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/search?view=azure-dotnet)

```

Comment:
Azure Cognitive Search

Reply:
copied this from the header in the doc page.  We should update that doc as well.

Reply:
I'm not sure where that page lives. @HeidiSteen -- The target of this link needs rebranding changes applied. Do you know how we go about doing that?